### DISCUSSION TITLE
Sunbird upgrade path from 4.4 to 5.x

### DISCUSSION THREAD

**Discussion by Renuka-S** [Sunbird-Ed/Community#361] [Documentation] [answered]
Hello Team,

Could you please share details or redirect me to the documentation for steps to be followed while doing the upgrade from 4.4 to the latest ex 5.0 or 5.1?

I am looking for recommended version upgrade steps as I assume we should not directly upgrade to 5.x from 4.4. There might be some middle versions that we should be upgrading to reach 5.x.

**Comment by Krishnaj20** [comment, marked as answer]
@gandham-santhosh @vinukumar-vs - for your attention.

**Reply by gandham-santhosh** [reply]
@Renuka-S there are no middle versions that you can upgrade directly from 4.4 . It is recommended to upgrade to the next following version. 4.5 -> 4.5, 4.5 -> 4.6, 4.6 -> 4.7 etc to make sure that there are no breaking changes. And it is recommended to verify/test after each minor version upgrade.

**Reply by Krishnaj20** [reply]
thanks @gandham-santhosh . Can you also share the doc links for these upgrades please?

**Reply by Krishnaj20** [reply]
@gandham-santhosh @Renuka-S - Is this query answered?

**Reply by gandham-santhosh** [reply]
@Renuka-S upgrade documentation till 4.10 can be found here http://docs.sunbird.org/4.5.0/developer-docs/upgrade/  and from 5.10 it can be found here http://docs.sunbird.org/4.5.0/developer-docs/upgrade/

**Reply by Krishnaj20** [reply]
@Renuka-S - is your query resolved?

**Reply by Krishnaj20** [reply]
@Renuka-S - marking this as answered since we haven't heard back from you.
